tensorflow/python/ops/signal/fft_ops.py

```diff
@@ -324,6 +325,66 @@ def _grad(op, grad):
 
   return _grad
 
+@tf_export("signal.fftshift")
+def fftshift(x, axes=None):
+    """Shift the zero-frequency component to the center of the spectrum.
+    This function swaps half-spaces for all axes listed (defaults to all).
+    Note that ``y[0]`` is the Nyquist component only if ``len(x)`` is even.
+
+    Args:
+      x: `Tensor`, input tensor.
+      axes: `int` or shape `tuple`, optional
+             Axes over which to shift.  Default is None, which shifts all axes.
+
+    Returns:
+      `Tensor`, The shifted tensor.
+
+    Example:
+      >>> fftshift([ 0.,  1.,  2.,  3.,  4., -5., -4., -3., -2., -1.])
+      <tf.Tensor: id=3, shape=(10,), dtype=float32,numpy=array
+      ([-5., -4., -3., -2., -1.,  0.,  1.,  2.,  3.,  4.], dtype=float32)>
+    """
+    x = _ops.convert_to_tensor_v2(x)
+    if axes is None:
+        axes = tuple(range(x.ndim))
+        shift = [dim // 2 for dim in x.shape]
+    elif isinstance(axes, int):
+        shift = x.shape[axes] // 2
+    else:
+        shift = [x.shape[ax] // 2 for ax in axes]
+
+    return manip_ops.roll(x, shift, axes)
+
+
+@tf_export("signal.ifftshift")
+def ifftshift(x, axes=None):
+    """The inverse of fftshift. Although identical for even-length x,
```

Comment:
Sorry, tiny nitpick :) The docstring should have a brief one-line description on the first line, then a blank space, then an optional longer description (where your sentence "Although..." could go).

See some of the math ops for examples:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/math_ops.py#L454-L480

It's important to match the format, because the documentation generator relies on it.

I think the "Example" section should also move above the Args section, I'm not certain what hte documentation generator will do with it.